src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -134,19 +134,13 @@
     <comment>To indicate Dynamo is up to date</comment>
```

Comment:
Hi @fanwgwg, how we usually update `Resources.en-US.resx` is first update `Resources.resx` with the desired modifications, then copy the file to overwrite `Resources.en-US.resx`, so both `Resources.en-US.resx` and `Resources.resx` should be completely identical. So please make sure you do that, too.